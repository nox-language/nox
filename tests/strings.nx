fun main() = (
    printi (ord "a");
    print (chr 97);
    print "\n";
    print (chr (ord "a"));
    print "\n";
    /*if "a" = "a" then // TODO: uncomment
        print "true\n";*/
    var char = getchar;
    print char;
    print "\n";
)
