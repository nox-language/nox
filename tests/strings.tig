(
    printi(ord("a"));
    print(chr(97));
    print("\n");
    print(chr(ord("a")));
    print("\n");
    if "a" = "a" then
        print("true\n");
    let var char := getchar()
    in (
        print(char);
        print("\n")
    )
    end
)
