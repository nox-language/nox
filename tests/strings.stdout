97
a
a
true
b
